packages/react-reconciler/src/ReactFiberLane.js

```diff
@@ -656,7 +677,8 @@ export function markRootSuspended(
 }
 
 export function markRootPinged(root: FiberRoot, pingedLanes: Lanes) {
-  root.pingedLanes |= root.suspendedLanes & pingedLanes;
+  // TODO: When would we ever ping lanes that we aren't suspended on?
+  root.pingedLanes |= pingedLanes;
```

Comment:
Need to investigate this further. Just transferring the updaters from `root.pingedLanes` to `updateLane` isn't enough. For some reason we're pinging when `root.suspendedLanes & pingedLanes === NoLanes`. Which seems odd intuitively but maybe the bug is `root.suspendedLanes` having no overlap with `pingedLanes`

Reply:
> When would we ever ping lanes that we aren't suspended on?

Maybe if we suspend on two Wakeables A and B, A gets pinged earlier, we unsuspend completely i.e. don't suspend on B again (e.g. conditional `use`?) and then we wouldn't want to retry when B gets pinged?